### PR TITLE
Connectors ETL

### PR DESCRIPTION
**Pull request by nataschaZima**
Implements ETL for Kafka, Spark and BI Connectors metrics (usage in Atlas). 

ETL only consists of extract and load stages (no transformation needed). 
The script can be run on any date/day of the week and will always give the following results: 
* For monthly trends, 6 months history including previous month
* For weekly trends, 8 weeks history including previous week

* Extract stage queries Athena and gets aggregated statistics
* Load stage inserts docs into Atlas cluster collections.

Resulting charts can be seen in this dashboard: 
https://charts.mongodb.com/charts-natalya-dtbfj/dashboards/2133e775-8c43-4d9d-b8b9-53a7ef9211e4

There are no options being passed to the script (yet). Can change based on Airflow implementation.

### CODE REVIEW

File: scripts/queries_connectors.py [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+        result = pd.read_sql(query,conn)
+        result = result.to_dict('records')
+        return result
+    except Exception as x:
+        print(x)
+        logging.error(x)
+        raise
+    finally:
+        print("closing connection")
+        conn.close()
```
Comment (Cartlidge) [inline scripts/queries_connectors.py:186]: I think this belongs in `etl_connectors.py`

File: scripts/queries_connectors.py [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+
+collections_queries = [
+    {'spark_monthly_new_vs_existing': query_spark_new_vs_existing_6_months()},
+    {'kafka_monthly_new_vs_existing': query_kafka_new_vs_existing_6_months()},
+    {'bic_active_clusters': query_bic_clusters()},
+    {'bic_usage_weekly': query_bic_weekly()},
+    {'kafka_weekly': query_kafka_weekly()},
+    {'spark_weekly': query_spark_weekly()},
+    {'kafka_sink_source_monthly': query_kafka_source_vs_sink_monthly()}
+]
```
Comment (Cartlidge) [inline scripts/queries_connectors.py:172]: Can you wrap this in a function like everything else?

File: scripts/queries_connectors.py [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+
+BIC_NAMES = [
+    'mongosqld',
+    'mongodrdl'
+]
+
+SOURCE_TABLE='cloud_backend_raw.dw__cloud_backend__rawclientmetadata'
+
+def driver_names_joined(names_list):
```
Comment (Cartlidge) [inline scripts/queries_connectors.py:26]: Can you add docstrings to all these functions?

File: scripts/etl_connectors.py [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+import dns
+from bson.objectid import ObjectId
+import datetime
+import pprint
+from datetime import timezone, datetime, tzinfo, date,time, timedelta
+import pdb
+#import logging
+import time
+from pyathenajdbc import connect
+from queries_connectors import *
```
Comment (Cartlidge) [inline scripts/etl_connectors.py:15]: Can you make this explicit, I think the only thing you reference is `collections_queries`

File: scripts/queries_connectors.py (hunk 1 above)
Comment (nataschaZima) [reply scripts/queries_connectors.py:186]: yeah you're right. Could be in a separate helpers file, but logically is closer to etl than queries file. will move it there.

File: scripts/queries_connectors.py (hunk 2 above)
Comment (nataschaZima) [reply scripts/queries_connectors.py:172]: done

File: scripts/queries_connectors.py (hunk 3 above)
Comment (nataschaZima) [reply scripts/queries_connectors.py:26]: done

File: scripts/etl_connectors.py (hunk 4 above)
Comment (nataschaZima) [reply scripts/etl_connectors.py:15]: done

**Pull request merged**